### DISCUSSION TITLE
Trouble with the Event library

### DISCUSSION THREAD

**Discussion by fijam** [SpenceKonde/megaTinyCore#644] [Q&A]
I'm trying to get the simple_event example to work on attiny412

```
#include <Event.h>
void setup() {
  Event2.set_generator(gen2::pin_pa1); // Set pin PA1 as event generator
  Event2.set_user(user::evouta_pin_pa2); // Set EVOUTA as event user
  Event2.start();
}

void loop() {
}
```

From what I understand, PA2 should follow the state of PA1. But if I pull PA1 up with an external resistor, PA2 stays at the ground level. Is there something missing?

**Comment by SpenceKonde** [comment]
What is PORTMUX.CTRLA set to? If it's not 1, that would be the problem. I don't have time to rigorousl test on all oparts, and these parts being as strange as they are, it is very possible that I missed the step of enabling it (that step is only required on 0 and 1 sreries parts, you see)
